azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/utilities/BottomCellAdapter.kt

```diff
@@ -15,9 +14,17 @@ internal class BottomCellAdapter(context: Context) : RecyclerView.Adapter<Bottom
     private var bottomCellItems: List<BottomCellItem> = mutableListOf()
 
     override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): BottomCellViewHolder {
-        val view: View =
-            mInflater.inflate(R.layout.azure_communication_ui_bottom_drawer_cell, parent, false)
-        return BottomCellViewHolder(view)
+        val bottomCellType = BottomCellItemType.values().get(viewType)
+        when (bottomCellType) {
+            BottomCellItemType.BottomMenuAction -> {
+                val view = mInflater.inflate(R.layout.azure_communication_ui_bottom_drawer_cell, parent, false)
+                return BottomCellActionViewHolder(view)
```

Comment:
These ViewHolder type names should be analyzed.

BottomCellActionViewHolder vs BottomCellViewHolder isn't really mentally clear.
BottomMenuCellActionViewHolder vs BottomMenuCellAvatarViewHolder (?)

Reply:
Included in aed9e65